experimental/add2app/example_module/pubspec.yaml

```diff
@@ -0,0 +1,30 @@
+name: example_module
+description: A simple module used to demo Flutter's Add2App features
+
+version: 1.0.0
+
+environment:
+  sdk: ">=2.4.0 <3.0.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+
```

Comment:
It would be nice to add at least one plugin to show how plugin registration works in SimpleIOSExample.

Reply:
+1.

I'm starting off with something basic, but at some point I'll go round to everyone working on add2app to make sure I'm demonstrating everything you feel should be covered.